department-of-veterans-affairs/caseflow
# 17513 pre-populate MST/PACT checkboxes if data is present in the model.

Resolves #((https://vajira.max.gov/browse/APPEALS-17513)

# Description
Pre-populates the MST/PACT checkbox on the add issues page if the data is present in the model. 

## Acceptance Criteria

## Testing Plan
1. Go to Intake and use one of the following veteran numbers.
- 992190636
- 107924012
- 329146737
- 981775045
- 738464961
- 245620198
- 955197769
2. Fill out the form with any options until you reach the add issues page.
3. Click the Add Issues button and select any of the listed issues.
4. Observe that you cannot change the value in the checkboxes and that they are already checked.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="751" alt="disabled autopop" src="https://user-images.githubusercontent.com/110805785/235526978-495fedd8-1a72-456b-9083-adac9839afe0.png">
<img width="747" alt="enabled autopop" src="https://user-images.githubusercontent.com/110805785/235526998-c9da9d4b-cf59-4f5f-9689-a18c75dadb98.png">


## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:
